spec/services/id_token_builder_spec.rb

```diff
@@ -3,16 +3,21 @@
 RSpec.describe IdTokenBuilder do
   include Rails.application.routes.url_helpers
 
+  let(:code) { SecureRandom.hex }
+
   let(:identity) do
     build(:identity,
           nonce: SecureRandom.hex,
           uuid: SecureRandom.uuid,
           ial: 3,
+          # this is a known value from an example developer guide
+          # https://developer.pingidentity.com/en/resources/openid-connect-developers-guide.html
+          access_token: 'dNZX1hEZ9wBCzNL40Upu646bdzQA',
```

Comment:
nice comment! could also name the string with a var name that makes this obvious. either way works!